Vagrantfile.example

```diff
@@ -64,13 +83,66 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         # vagrant destroy and vagrant up to get a new one).
         #
         # domain.volume_cache = "unsafe"
+    
+        # Uncomment this to resize the root partition and filesystem to fill the base box disk
+        # This script is only guaranteed to work with the default official fedora image, and is
+        # only needed it you changed machine_virtual_size above.
+        # For other boxen, use at your own risk
+        # override.vm.provision "shell", path: "playpen/vagrant-resize-disk.sh"
+    end
+
+    dev.vm.provider :docker do |d, override|
+        override.vm.box = nil
+        d.image = 'rohanpm/fedora-vagrant:23'
```

Comment:
I'd rather use the "official" fedora image directly, but looking at the dockerfile here that doesn't seem reasonable.